index.html

```diff
@@ -1,51 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Collins Akorda - Portfolio</title>
   <link rel="stylesheet" href="style.css">
 </head>
-
 <body>
-
   <header>
     <nav>
       <div class="nav-bar">
         <a href="#" class="logo">
           <img class="menuIcone" src="./images/logo.png" alt="logo Image">
         </a>
-
-        </button-->
         <input type="checkbox" class="side-menu" id="side-menu" />
-        <label for="side-menu" class="hamb">
+        <label id="btn" for="side-menu" class="hamb">
           <span class="hamb-line"></span>
         </label>
-        <div class="nav">
+        <div id="nav" class="nav">
           <ul class="menu">
-            <li><a href="#Portfolio">Portfolio</a></li>
-            <li><a href="#About">About</a></li>
-            <li><a href="#contact">Contacts</a></li>
```

Comment:
- [ ] I can see that you added some of the required changes by the previous reviewer good job so far 👍 but there is something you are still missing as requested by the previous reviewer kindly consider making sure that when someone clicks any of the links the whole menu dissapear menu but in your case it's still not disappearing kindly fix it check the image below to easily understand it 👇 
- 
<img width="385" alt="Screen Shot 2023-04-19 at 17 22 36" src="https://user-images.githubusercontent.com/60146030/233124703-f5ac2b82-9a1e-4c05-ae10-b5ba37cb5e69.png">

- [ ]  last but not least kindly consider making sure that your mobile menu looks like the one given in the design make sure that the blue background covers the whole page even the header to have an exact duplicate of the provided design check the images on this [link ](https://github.com/firstpj/Portfolio-Mobile-first-set-up/pull/9#discussion_r1169922670) for more clarification